core/src/core.py

```diff
@@ -98,14 +111,34 @@ def start(self):
 		plugins.__path__ = []
 		plugins.__path__.append (os.path.join(rootPath, 'src', ControllerType))
 
+		moduleList = {}
```

Comment:
Does this list contains all loaded modules? If so, I suggest change the name to "loadedModules"